crates/rowan-parser/src/lib.rs

```diff
@@ -1,2 +1,32 @@
+use syntax::SyntaxNode;
+
 pub mod parser;
 pub mod syntax;
+
+pub fn ast_printer(root: SyntaxNode, level: usize, compatible_with_postcss: bool) {
```

Comment:
if compatible_with_postcss is false, should print all the SyntaxNode include, `Prop`, `Selector` ..., which postcss-js don't have.